src/Connection/Connection.php

```diff
@@ -1029,11 +981,11 @@ public function setShuffleMasters(bool $value): void
      * one of these configurations will be chosen and used to create a DB connection for performing read queries only.
      *
      * @param string $key index slave connection.
-     * @param array $config The configuration that should be merged with every slave configuration
+     * @param ConnectionInterface $slave The connection every slave.
      */
-    public function setSlaves(string $key, array $config = []): void
+    public function setSlaves(string $key, ConnectionInterface $slave): void
```

Comment:
The method has different meaning now. We were able to provide an array of slave connections and a key that points to main one. Now we're able to provide only a single connection.

Reply:
Not at all can you provide several connections the difference is that now it is an array of connections, before you could only specify the configuration of the connection, it is the same the only thing that stops the connection already created. 

Before PR:
https://github.com/yiisoft/db-sqlite/blob/db954c66f6f87cbf383ba64b57c1224eea602157/tests/ConnectionTest.php#L446

After PR: 
https://github.com/terabytesoftw/db-sqlite/blob/b6fc05fe8462810410d49b5a724141cbedb07d63/tests/ConnectionTest.php#L419

Reply:
fixed.